types/spotify-api/index.d.ts

```diff
@@ -672,7 +669,7 @@ declare namespace SpotifyApi {
      * GET /v1/tracks/{id}
      * https://developer.spotify.com/web-api/get-track/
      */
-    interface SingleTrackResponse extends TrackObjectFull {}
+    interface SingleTrackResponse extends TrackObjectFull { }
```

Comment:
I presume these are auto-formatted? If so, they aren't strictly part of this PR, but including them would make future PRs easier, so I think you can keep them.